### ISSUE TITLE
Improve MacOS support.

### ISSUE THREAD

**Issue by dsm** [ra3xdh/qucs_s#797]
This issue was created to keep track of improvements and issues in one place. 

- [x] fix "About qucs-s" dialog and "About Qt". #799  
- [x] add MacOS arm64 and x86_64 release to CI. #799

**Comment by dsm** [comment]
@ra3xdh  In qucs-init.cpp start line after 666 disabled for macos and About qucs-s run as expected but why I didn't understand. 

```
#ifdef __APPLE__
  helpAboutQt = new QAction(tr("About Qt..."), this);
  helpAboutQt->setStatusTip(tr("About Qt"));
  helpAboutQt->setWhatsThis(tr("About Qt\n\nAbout Qt by Trolltech"));
  connect(helpAboutQt, SIGNAL(triggered(bool)), qApp, SLOT(aboutQt()));
#endif
```
I also disabled this in the same file.
```
#ifndef __APPLE__ 
  helpMenu->addAction(helpAboutQt);
#endif
```

**Comment by dsm** [comment]
I almost finished new macos ci, About qucs-s dialog fixed. and examples, library and symbols folder added. Please test this release and give feedback, thanks. 

[qucs_s_24.2.1_arm64_.dmg](https://github.com/dsm/qucs_s/releases/download/continuous_build/qucs_s_24.2.1_arm64_.dmg)

**Comment by ra3xdh** [comment]
Thanks! Let's wait for the feedback, then please prepare a patch.

**Comment by dsm** [comment]
patch is on the way, can you update qucsator_rf submodule.

**Comment by ra3xdh** [comment]
I have just updated qucsator_rf by this commit https://github.com/ra3xdh/qucs_s/commit/6e6fb4fb70fd90d3ed8e9b171fb6673f95ceb222

**Comment by dsm** [comment]
#799 fixes prepared.

**Comment by bnilsson11** [comment]
Sorry, I cannot find [qucs_s_24.2.1_arm64_.dmg](https://github.com/dsm/qucs_s/releases/download/continuous_build/qucs_s_24.2.1_arm64_.dmg)
I get a 404 when clicking on it.
Where should I look?

**Comment by dsm** [comment]
macos release patch is merge to main repo you can look to release section.

**Comment by bnilsson11** [comment]
Ok, got it.
About.. is fine.
No System Libraries.

**Comment by dsm** [comment]
I don't know, when I tried yesterday there is no problem with system libraries

**Comment by dsm** [comment]
sorry in ci there should be mistakenly change folder name 'share' to 're' and didn't work

**Comment by bnilsson11** [comment]
True. I changed `re` to `share` and now it is working.

**Comment by dsm** [comment]
latest commit I changed mistakenly :) this issue fixed in #800.
